app.js

```diff
@@ -44,6 +45,20 @@ const passportConfig = require('./config/passport');
  */
 const app = express();
 
+/** 
+ * rate-limiters for /login and /signup POST requests
+ */
+const loginLimiter = rateLimit({
+  // block IP for 15 minutes after 20 requests
+  windowMs: 15 * 60 * 1000,
+  max: 20
+});
+const signupLimiter = rateLimit({
+  // block IP for 1 hour after 5 requests
+  windowMs: 60 * 60 * 1000,
```

Comment:
1 hour is too much in my opinion ... we need to remember some users could have multiple problems during this phase (invalid pass, mail already registered etc ...)

Reply:
@Mik317 what about 1hour after 15requests?

Reply:
IMHO 15 requests every 15 min might be a good thing to keep. But to be 100% sure I think we should ask the project dev about this time limit.